src/App.tsx

```diff
@@ -1,24 +1,143 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Filter } from './components/Filter';
+import { Notifications } from './components/Notifications';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { ErrorText } from './types/ErrorText';
+import { Filters } from './types/Filters';
+
+const USER_ID = 11213;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loadedTodos, setLoadingTodos] = useState<number[]>([]);
+  const [errorMessage, setErrorMessage] = useState(ErrorText.Empty);
+  const [selectedFilter, setSelectedFilter] = useState<Filters>(Filters.All);
+  const [title, setTitle] = useState('');
+  const [disabledInput, setDisabledInput] = useState(false);
```

Comment:
Rename all boolean variable if present using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isDisabledInput, setIsDisabledInput] = useState(false);
```